ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
thanks for reviewing, that's a fair point

maybe I could leave the anon user with writepermissions=false for the sad path + setup a different, write-enabled user for the happy path?